README.md

```diff
@@ -1,16 +1,81 @@
-# AceBook
+# Acebook
 
-Fork this repository to `acebook-teamname` and customize
-the below**
+## Summary
 
-[You can find the engineering project outline here.](https://github.com/makersacademy/course/tree/master/engineering_projects/rails)
+Week 8 and 9 Makers Academy engineering project to develop an Facebook clone, called Acebook. <br>
```

Comment:
typo -
should be '... a Facebook clone...' 
and not '... an Facebook clone... '

Reply:
Thanks - updated